torch/csrc/jit/passes/create_autodiff_subgraphs.cpp

```diff
@@ -174,6 +188,11 @@ class SubgraphSlicer {
     if (node->kind() == prim::Constant) {
       return false;
     }
+    // view ops as outputs of differentiable subgraphs can cause incorrect differentiation
+    // for now, do not include them in the subgraph
```

Comment:
This should make clang-format happy
```suggestion
    // view ops as outputs of differentiable subgraphs can cause incorrect
    // differentiation for now, do not include them in the subgraph
```